src/index.html

```diff
@@ -22,6 +22,455 @@
+
+          <div class="card__botoom-stars">
+            <div class="stars stars--4">
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+            </div>
+
+            <p class="card__botom-text">Reviews: 5</p>
+          </div>
+
+          <div class="card__bottom-cost">
+            <div class="card__bottom-price">Price:</div>
+            <div class="card__bottom-cost">$2,199</div>
+          </div>
+          <a
+            class="card__bottom-button"
+            data-qa="hover"
+            href="#"
+          >
+            BUY
+          </a>
+        </div>
+      </article>
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__top">
+          <img
+            src="images/imac.jpeg"
+            class="card__top-img"
+            alt="imac"
+          />
+        </div>
+
+        <div class="card__bottom">
+          <h2 class="card__bottom-tittle">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__bottom-code">Product code: 195434</p>
+
+          <div class="card__botoom-stars">
+            <div class="stars stars--4">
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+            </div>
+
+            <p class="card__botom-text">Reviews: 5</p>
+          </div>
+
+          <div class="card__bottom-cost">
+            <div class="card__bottom-price">Price:</div>
+            <div class="card__bottom-cost">$2,199</div>
+          </div>
+          <a
+            class="card__bottom-button"
+            data-qa="hover"
+            href="#"
+          >
+            BUY
+          </a>
+        </div>
+      </article>
+      <article
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__top">
+          <img
+            src="images/imac.jpeg"
+            class="card__top-img"
+            alt="imac"
+          />
+        </div>
+
+        <div class="card__bottom">
+          <h2 class="card__bottom-tittle">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__bottom-code">Product code: 195434</p>
+
+          <div class="card__botoom-stars">
+            <div class="stars stars--4">
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+              <div class="stars__item"></div>
+            </div>
+
+            <p class="card__botom-text">Reviews: 5</p>
```

Comment:
There's a typo in the class name 'card__botom-text'. It should be 'card__bottom-text'.